udata/core/site/models.py

```diff
@@ -28,6 +30,112 @@ class Site(WithMetrics, db.Document):
 
     def __str__(self):
         return self.title or ''
+    
+    def count_users(self):
+        from udata.models import User
+        self.metrics["users"] = User.objects.count()
```

Comment:
Filtering on activated users might be a good idea (`confirmed_at__ne=None`)

Reply:
\+ `deleted=None`

Reply:
(there might be a `visible()` method on User.objects too)

Reply:
The visible method checks if the user has any dataset or reuse to return True. I don't think it is relevant in this metric. We might want to know someday if there is a huge load of random users created suddenly.